decidim-surveys/app/views/decidim/surveys/admin/publish_answers/index.html.erb

```diff
@@ -0,0 +1,44 @@
+
+<% add_decidim_page_title(t(".title")) %>
+<div class="item_show__header border-none">
+  <div class="w-full">
+    <h1 class="item_show__header-title">
+      <%= t(".title") %>
+      <%= link_to t("actions.back", scope: "decidim.forms.admin.questionnaires"), questionnaire_url, class: "button button__sm button__secondary new" %>
```

Comment:
This URL is not ok. Clinking it would show this page: 
![image](https://github.com/user-attachments/assets/f95bf30a-07fa-46cc-92ca-2a8e2e2f3ab3)

Reply:
Fixed on [bfcf08c](https://github.com/decidim/decidim/pull/13786/commits/bfcf08c79d6ceb7c2f9cd3875a6c7a416a696b4d). I think this changed after the "multiple surveys from the same component" was merged (or I just messed up)

Reply:
Doesn't really matter. Is important that we've caught it in time.